sites/markdown-editor/src/App.jsx

```diff
@@ -12,14 +12,19 @@ function sendNotification(notification) {
   }
 }
 
-let initialValue = '';
+let initialValue = 'oiawhdoahw';
+let initialEditableValue = true;
```

Comment:
I think `editable` is most likely not going to change during the lifetime of an individual webview, so I think it'd be better to set this directly on `window`, like you do with `window.THEME`. That will ensure we won't get a flash of the editor loading for a second.

Reply:
good idea